src/storage-controller/src/instance.rs

```diff
@@ -65,6 +62,8 @@ pub(crate) struct Instance<T> {
     now: NowFn,
     /// Responses ready for delivery on the next call to [`Instance::recv`].
     ready_responses: VecDeque<StorageResponse<T>>,
+    /// A receiver for responses from replicas.
+    instance_response_tx: mpsc::UnboundedSender<StorageResponse<T>>,
```

Comment:
Nit: I'd name this just `response_tx`, since we're already in the context of an instance here.